lisa/operating_system.py

```diff
@@ -705,13 +706,15 @@ def add_repository(self, repo: str, key_location: str = "") -> None:
             expected_exit_code_failure_message="fail to add repository",
         )
 
+    @retry(tries=60, delay=5)  # type: ignore
```

Comment:
Let's try 10 times firstly, 60 times not only 60 * 5. Because the install command also takes long time. If it's too long, it's too long when troubleshooting.

Reply:
split it into another PR